src/panels/lovelace/common/handle-action.ts

```diff
@@ -17,6 +17,7 @@ declare global {
 export type ActionConfigParams = {
   entity?: string;
   camera_image?: string;
+  image_entity?: string;
```

Comment:
I wonder if we should only pass entity and let the caller do `entity: image_entity`

Reply:
Maybe, I just continued the same way we did it for camera 🤷‍♂️